docs/uninstall/conda.rst

```diff
@@ -11,25 +11,13 @@ This guide will help you uninstall Conda from your computer. This can be useful
 
 * You have issues with packages.
 
+* You already have the Anaconda distribution installed, but need to switch to miniconda. 
```

Comment:
Title: `Miniconda`

Reply:
What do you mean with this? 'Title: Miniconda'